### PR TITLE
feat(molecule/avatar): add new sizes

### PR DESCRIPTION
**Pull request by xaviermariaconejo**
## MOLECULE/AVATAR
**TASK**: [ISSUE 1329](https://github.com/SUI-Components/sui-components/issues/1329)

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Only
- [ ] Refactor

### Description, Motivation and Context
The issue: [1239](https://github.com/SUI-Components/sui-components/issues/1329) is about add more sizes to the avatar, so it has done

### Screenshots - Animations

image

### CODE REVIEW

File: components/molecule/avatar/src/AvatarBadge/index.js [hunk 1]
```diff
@@ -12,10 +12,12 @@ export const AVATAR_BADGE_PLACEMENTS = {
 }
 
 export const AVATAR_BADGE_SIZES = {
+  XXLARGE: 'xxlarge',
   XLARGE: 'xlarge',
   LARGE: 'large',
   MEDIUM: 'medium',
-  SMALL: 'small'
+  SMALL: 'small',
+  XSMALL: 'ssmall'
```
Comment (PablitoGS) [inline components/molecule/avatar/src/AvatarBadge/index.js]: there is mistake here

**Review by andresz1 — APPROVED** [review]

**Comment by andresin87** [comment]
[https://sui-components-bwpkmx1le.vercel.app/workbench/molecule/avatar/demo](https://sui-components-bwpkmx1le.vercel.app/workbench/molecule/avatar/demo)

**Review by andresin87 — CHANGES_REQUESTED** [review]
fix the typo. THKS!

**Review by PablitoGS — APPROVED** [review]

File: components/molecule/avatar/src/AvatarBadge/index.js (hunk 1 above)
Comment (xaviermariaconejo) [reply components/molecule/avatar/src/AvatarBadge/index.js]: Fixed

**Review by turolopezsanabria — COMMENTED** [review]
👏

**Comment by andresin87** [comment]
👍

**Pull request merged**